mix.exs

```diff
@@ -35,6 +35,9 @@ defmodule ChatApi.MixProject do
     [
       {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
+      {:hackney, "~> 1.9"},
```

Comment:
what's this dependency for btw?

Reply:
Its a dependency required for Swoosh the email client it yells at us if we don't have it